spec/spec_helper.rb

```diff
@@ -93,6 +93,9 @@
   # Ensure we start with consistent config settings
   config.before(:each) { Spree::Config.products_require_tax_category = false }
 
+  # Tests assume AUD is the default currency.
+  config.before(:each) {Spree::Config.currency = "AUD" }
```

Comment:
Is there any way to set this once instead of per each example? `before(:each)` is time and resource consuming.